test/integration/contractor_can_logout_test.rb

```diff
@@ -11,9 +11,11 @@ class ContractorCanLogoutTest < ActionDispatch::IntegrationTest
       click_on "Contractor Login"
     end
 
-    assert page.has_content?("Logged in: Contractor #{contractor.first_name}")
+    assert page.has_content?("View Account: #{contractor.first_name}")
 
-    click_on "Logout"
+    within '.container' do 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.